include/eve/module/core/regular/impl/simd/arm/neon/any.hpp

```diff
@@ -55,14 +55,22 @@ any_(EVE_SUPPORTS(neon128_),
   else if constexpr( !C::is_complete ) return any_(EVE_RETARGET(cpu_), cond, v0);
   else if constexpr( eve::current_api >= eve::asimd )
   {
-    // There is no vmaxvq_u64, so we use vmaxvq_u32
-    auto dwords = eve::bit_cast(v0, eve::as<u32_4> {});
-    return vmaxvq_u32(dwords);
+    // Adapted from https://github.com/dotnet/runtime/pull/75864
+    auto mask = bit_cast(v0.bits(), as<u32_4>{});
+    return bit_cast(u32_4(vpmaxq_u32(mask,mask)), as<u64_2>()).get(0) != 0;
   }
-  else // chars, no asimd
+  else // no asimd
   {
-    auto dwords = eve::bit_cast(v0, eve::as<u32_4>());
-    return eve::any[ignore_none](detail::to_logical(dwords));
+    auto mask = eve::bit_cast(v0, eve::as<u32_4>());
+    if constexpr( sizeof(T) > 2 )
+    {
+      // Adapted from https://stackoverflow.com/a/41007755/737268
+      return vget_lane_u64(vreinterpret_u64_u16(vqmovn_u32(mask)), 0) != 0;
```

Comment:
We specifically not did this - because there is no 64 bit numbers. This a no asimd usecase.
Same as the other.

Reply:
Rolled back